brownie_tokens/forked.py

```diff
@@ -131,7 +131,7 @@ def mint_0xfE18be6b3Bd88A2D2A7f928d00292E7a9963CfC6(
 ) -> None:
     # Synth sBTC
     target_contract = Contract("0xDB91E4B3b6E19bF22E810C43273eae48C9037e74")
-    target_contract.issue(target, amount, {"from": "0x778D2d3E3515e42573EB1e6a8d8915D4a22D9d54"})
+    target_contract.issue(target, amount, {"from": "0xB774711F0BC1306ce892ef8C02D0476dCccB46B7"})
```

Comment:
This change needs to be applied to every synth's custom mint logic.  Also, maybe we should handle it in a way such that we don't have to update again when they deploy a new `Exchanger` implementation.

A private function to get the current `Exchanger` from the `AddressResolver` would work:

```python
def _snx_exchanger():
    abi = [{
        "inputs": [
            {
                "name": "name",
                "type": "bytes32"
            }
        ],
        "name": "getAddress",
        "outputs": [
            {
                "internalType": "uint256",
                "name": "",
                "type": "address"
            }
        ],
        "stateMutability": "view",
        "type": "function"
    }]
    resolver = Contract.from_abi("AddressResolver", "0x4E3b31eB0E5CB73641EE1E65E7dCEFe520bA3ef2", abi)
    return resolver.getAddress("0x45786368616e6765720000000000000000000000000000000000000000000000")
```